### PR TITLE
Fb ag gd branch 2

### PR DESCRIPTION
**Pull request by aleksgor**
YAHOO-119 and YAHOO-120 fix coverage 
org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler 
and
org.apache.hadoop.yarn.server.nodemanager.LinuxContainerExecutor

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@
 target
 hadoop-hdfs-project/hadoop-hdfs/downloads
 hadoop-hdfs-project/hadoop-hdfs-httpfs/downloads
+hadoop-common-project/hadoop-common/build
+hadoop-hdfs-project/hadoop-hdfs/build
+hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/.clover
```
Comment (dennisyv) [inline .gitignore]: do we really need these ignores?

Comment (aleksgor) [inline .gitignore]: No. But it more comfortable for me

Gorshkov Aleksey

Grid Dynamics
 +7 952 203 9796
Skype:aleksgorpsk
www.griddynamics.com
email: aleksgor@gmail.com

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestLinuxContainerExecutor.java [hunk 2]
```diff
@@ -257,4 +269,80 @@ public void run() {
...
+	  LinuxContainerExecutor executer= new LinuxContainerExecutor();
+	  executer.setConf(config);
+	  executer.init();
+	  
+	  // try to get exception
+	   config= new Configuration();
+	  config.set("yarn.nodemanager.linux-container-executor.path", "/bin/sh");
+	  executer= new LinuxContainerExecutor();
+	  executer.setConf(config);
+	  executer.init();
```
Comment (ikatsov) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestLinuxContainerExecutor.java]: is it possible to add assertions?

File: .gitignore (hunk 1 above)
Comment (dennisyv) [inline .gitignore]: please don't commit this stuff - it will surely bring us to merges

**Comment by diggerk** [comment]
How about using task ID for branch names? I guess in this case it would be YAHOO-119

**Pull request closed without merging**